renderdoc/driver/vulkan/wrappers/vk_misc_funcs.cpp

```diff
@@ -1699,6 +1705,8 @@ void WrappedVulkan::vkResetQueryPool(VkDevice device, VkQueryPool queryPool, uin
   SERIALISE_TIME_CALL(
       ObjDisp(device)->ResetQueryPool(Unwrap(device), Unwrap(queryPool), firstQuery, queryCount));
 
+  GetRecord(queryPool)->queryPoolInfo->Reset(firstQuery, queryCount);
```

Comment:
This also misses handling of `CmdResetQueryPool` but I think with the other comments handling reset may not be needed at all.